### DISCUSSION TITLE
Problems to use "addon-storysource"

### DISCUSSION THREAD

**Discussion by gabruw** [storybookjs/storybook#11100] [Help] [answered]
Hello! 

Recently I have been studying the Storybook and their addons. In my solution, I wanna show the source code from the component. For it, I'm using the "addon-storysource" but in my solution, always shows the source code of the component that's in MDX file. Is it possible that source code of the component itself is shown? I'm using the "6.0.0-beta.23" version:

[package.json]
![image](https://user-images.githubusercontent.com/36605990/84150991-0c091100-aa39-11ea-987b-4dbb239fca72.png)

[main.js]
![image](https://user-images.githubusercontent.com/36605990/84150953-fbf13180-aa38-11ea-874f-40010654ce26.png)

[preview.js]
![image](https://user-images.githubusercontent.com/36605990/84151025-19be9680-aa39-11ea-91d2-e2e1ca80f0a1.png)

[MDX example]
![image](https://user-images.githubusercontent.com/36605990/84151117-42df2700-aa39-11ea-9d99-47c9a5f2e00b.png)

[Result in Browser]
![image](https://user-images.githubusercontent.com/36605990/84151167-57232400-aa39-11ea-8989-db7c76a1f034.png)

**Comment by shilman** [comment, marked as answer]
Storysource doesn't work well with MDX. This is probably better suited as an issue rather than a discussion.

**Reply by gabruw** [reply]
Thx @shilman , I'll do this.
